### ISSUE TITLE
Service Request: Add LinkedIn Darkmode

### ISSUE THREAD

**Issue by Julian88Tex** [MikeDabrowski/franz-themed-recipes#6]
Service Request: Add LinkedIn Darkmode

Attempt was made on another repo but supposedly there may be a Franz bug preventing it: https://github.com/ducfilan/Dark-mode-Franz/issues/6

**Comment by MikeDabrowski** [comment]
https://github.com/MikeDabrowski/franz-themed-recipes/commit/44b6e663570bc4098f8555f9533e7128d548f532

Check now. I grabbed styles from userstyles. That is all I can do for now due to lack of time before holidays. If you find some bright elements that should not be bright, and can fix it, then create PR with changes.

**Comment by Julian88Tex** [comment]
@MikeDabrowski Thanks for doing this... I'm still having trouble though getting it to work. I haven't tried with your fork yet so I'll do that next. By any chance do you have a screenshot of it in action?

**Comment by MikeDabrowski** [comment]
![image](https://user-images.githubusercontent.com/17836911/70791018-7a289000-1d96-11ea-9044-89b31b5799de.png)

I don't know how you load css without my fork so Im not really able to help there. My css works when you add theme-dark class to body of service.

**Comment by Julian88Tex** [comment]
Makes sense and thanks for the pic. I'll let you know if I get it to work.
